src/app/modules/verification/__tests__/verification.service.new.spec.ts

```diff
@@ -229,4 +254,246 @@ describe('Verification service', () => {
       expect(result._unsafeUnwrapErr()).toBeInstanceOf(DatabaseError)
     })
   })
+
+  describe('sendNewOtp', () => {
+    let updateHashSpy: jest.SpyInstance<
+      Promise<IVerificationSchema | null>,
+      [updateData: UpdateFieldData]
+    >
+
+    beforeEach(() => {
+      updateHashSpy = jest
+        .spyOn(VerificationModel, 'updateHashForField')
+        .mockResolvedValueOnce(mockTransaction)
+      MockSmsFactory.sendVerificationOtp.mockReturnValueOnce(okAsync(true))
```

Comment:
Same here, can just do a mockReturnValue in a beforeAll (unless they will get reset somehow)